test/dotnet-new.Tests/GivenThatIWantANewCSLIbrary.cs

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Microsoft.DotNet.Tools.Test.Utilities;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using Xunit;
+using FluentAssertions;
+
+namespace Microsoft.DotNet.Tests
+{
+    public class GivenThatIWantANewCSLibrary : TestBase
+    {
+        
+        [Fact]
+        public void When_library_created_Then_project_restores()
+        {
+            var rootPath = Temp.CreateDirectory().Path;
+            var projectJsonFile = Path.Combine(rootPath, "project.json");
+
+            new TestCommand("dotnet") { WorkingDirectory = rootPath }
+                .Execute("new --type lib");
```

Comment:
No new Fact.  Just add the ".Should().Pass()" to the end of this ".Execute()" call.  That way if it fails, we catch it here, and we know what first started failing.